modules/installation-configuration-parameters.adoc

```diff
@@ -927,10 +927,22 @@ Additional Azure configuration parameters are described in the following table:
 |====
 |Parameter|Description|Values
 
+|`compute.platform.azure.osDisk.diskSizeGB`
+|The Azure disk size for the VM.
+|Integer that represents the size of the disk in GB. The minimum supported disk size is `1024`.
```

Comment:
in  [Table 4. ](https://docs.openshift.com/container-platform/4.10/installing/installing_azure/installing-azure-customizations.html#installation-configuration-parameters-additional-azure_installing-azure-customizations)controlPlane.platform.azure.osDisk.diskSizeGB , the default value is 1024,  compute.platform.azure.osDisk.diskSizeGB the default value is 128, not the "minimum supported"

ref https://docs.openshift.com/container-platform/4.10/installing/installing_azure/installing-azure-customizations.html#installation-minimum-resource-requirements_installing-azure-customizations

Reply:
Thank you for adding that! Just updated!